velox/dwio/common/FlatMapHelper.h

```diff
@@ -67,6 +67,7 @@ void initializeFlatVector(
   if (!vector) {
     vector = std::make_shared<FlatVector<T>>(
         &pool,
+        CppToType<T>::create(),
```

Comment:
would the idea be to remove CppToType altogether, or that we would still keep it around for test usages like this?

Reply:
Yes, it would be nice to remove CppToType<T> template, but if not, then move it to test-only code so it cannot be used in production code by accident.